apps/prairielearn/elements/pl-file-upload/pl-file-upload.mustache

```diff
@@ -22,6 +25,19 @@ $(() => {
   </div>
   {{/editable}}
   <div class="messages">
+{{#parse_error}}
+    <a role="button"
+      class="btn btn-light border d-flex align-items-center text-danger"
+      data-toggle="popover"
+      data-html="true"
+      title="Format Error"
+      data-placement="auto"
+      data-trigger="focus"
+      tabindex="0"
+      data-content="{{.}}">
+      <span class="mr-1">Invalid </span> <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+    </a>
+{{/parse_error}}
```

Comment:
Could you post a screenshot of how this error looks / where it comes up? Just asking because different elements display the parse error slightly differently. Also, it looks like this error is being put next to the element, so that would resolve this issue: https://github.com/PrairieLearn/PrairieLearn/issues/5902

Reply:
I will spend a bit more time on re-working this, because at the moment there is basically one big box that shows all the file-related errors. The previous state was basically no visible error with any feedback (other than the "invalid" flag), so I mostly just wanted to make it visible what went wrong, especially with the other changes that make it a bit less obvious why a file upload submission might be invalid.

Reply:
This is how it looks now. I think this is relatively consistent to other elements. 

<img width="803" alt="Screenshot 2024-08-05 at 1 50 27 PM" src="https://github.com/user-attachments/assets/04b54066-e946-494e-9083-71d222c93efe">

Reply:
I think it looks good!